_config/products.yml

```diff
@@ -78,6 +78,33 @@ devstudio:
         update_site_url: https://devstudio.redhat.com/9.0/development/updates/
         #marketplace_url: https://marketplace.eclipse.org/node/1616973
         #marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2015-02-02
+        supported_devstudio_is_version: 9.0.0.Alpha2
```

Comment:
A little bit late for 9.0.0.Alpha2, no? ;-)
And please fix the release date too.